openapi/openapi.yaml

```diff
@@ -88,7 +88,35 @@ paths:
         '404':
           $ref: "#/components/responses/NotFound"
         '500':
-          $ref: "#/components/responses/InternalServerError" 
+          $ref: "#/components/responses/InternalServerError"
+  /capital-projects/{managingCode}/{capitalProjectid}/capital-commitments:
+    get:
+      summary: Find capital commitments associated with a specific capital project
+      operationId: findCapitalCommitmentsByManangingCodeCapitalProjectId
```

Comment:
Spelling `Mananging`

Reply:
that what I get for copy/pasting

😩

Reply:
Lol, I just checked the original ticket. I'm blaming my keyboard for the typo in the ticket